plugins/Live/tests/System/ApiTest.php

```diff
@@ -78,6 +78,19 @@ public function getApiForTesting()
             )
         );
 
+        $apiToTest[] = array(array('Live.getLastVisitsDetails'),
```

Comment:
added this test to ensure it works besides the already existing tests that generates the SQL where we can see in the SQL query that it will now work